### DISCUSSION TITLE
1543030 《仙剑七》游戏本体+DLC《人间如梦》

### DISCUSSION THREAD

**Discussion by a1144940203** [pjy612/SteamManifestCache#98] [Share]
[《仙剑七》游戏本体+DLC《人间如梦》.txt](https://github.com/pjy612/SteamManifestCache/files/13717379/%2BDLC.txt)
[《仙剑七》游戏本体+DLC《人间如梦》.zip](https://github.com/pjy612/SteamManifestCache/files/13717384/%2BDLC.zip)

内测码 ：F449D131C87B7ED6
邮箱 1144940203@qq.com

**Comment by pjy612** [comment]
查收

**Comment by a1144940203** [comment]
@pjy612  dalao
